### ISSUE TITLE
Ableton Live 11 drag and drop.

### ISSUE THREAD

**Issue by DarkAlchy** [essej/paulxstretch#21]
Drag a sample from the Ableton samples folder to the VST changes the cursor but the sample is never loaded into the vst.

**Comment by essej** [comment]
So you are dragging from the area within Ableton itself? If you drag from the system’s native file browser (finder, file explorer, etc) does it work OK?

**Comment by DarkAlchy** [comment]
Yes, with the vst3, or the standalone I can drag and drop via Windows, but from Ableton Live's I cannot.  I saw a video and they dragged then dropped from within FL Studio.  I just now tried it in FL Studio and the drag and drop from its browser does work just not in Ableton Live 11.

**Comment by DarkAlchy** [comment]
I figured out why as it just struck me what is happening thanks to trying it in FL Studio and that is the samples in Ableton are all .aif format whereas in FL Studio they come with .wav.  I tried a sample I have in the browser of .wav and it opened.  This needs to be able to handle the .aif file format.
